### PR TITLE
Add Sky Mobile gradient

### PR DESCRIPTION
**Pull request by steallan-sky**
## Description
Adds the new gradient for Sky Mobile to the gradient set.

## Related Issue
connects to https://github.com/sky-uk/skymobile-service/issues/2162

## Motivation and Context
This is needed so that we can call this gradient in a new pages-lib component and also so that we no longer need to map merge the Sky Mobile gradient into the main set.

## How Has This Been Tested?
Ran Toolkit preview and added the relevant class to a tile to see the gradient applied (see screenshots).

## Screenshots
image 1

## Types of Changes

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [ ] Internal *(framework-only)*
- [ ] Bug Fix *(non-breaking change which fixes an issue)*
- [x] New Feature *(non-breaking change which adds functionality)*
- [ ] Breaking change *(fix or feature that would cause existing functionality to change)*

## Browser Support

- [ ] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] IE9
- [ ] IE10
- [ ] IE11
- [ ] Opera
- [ ] Safari
- [ ] Mobile Devices
- [ ] Screen-readers (e.g. JAWS, Apple VoiceOver)

## Checklist

- [x] All **design and code** conforms to the [Design Contribution Guidelines](https://github.com/sky-uk/toolkit/blob/develop/CONTRIBUTING.md#design-contributions).
- [ ] New functions and mixins have appropriate tests.
- [x] All new and existing tests passed.
- [ ] I have added instructions on how to test my changes.

### CODE REVIEW

File: packages/sky-toolkit-core/settings/_gradients.scss [hunk 1]
```diff
@@ -58,6 +58,10 @@ $gradients: (
     0%:   #68879c,
     100%: #67a0b5
   ),
+  sky-mobile: (
```
Comment (lukewhitehouse) [inline packages/sky-toolkit-core/settings/_gradients.scss]: Need to know if this is for both sales and service, or just service. Will try chasing Mr Gregory

Comment (lukewhitehouse) [reply packages/sky-toolkit-core/settings/_gradients.scss]: Just thinking actually this may cause problems with our gradient mapping, will check tomorrow

Comment (steallan-sky) [reply packages/sky-toolkit-core/settings/_gradients.scss]: According to Joy, this gradient is the official mobile brand gradient.

Comment (MrDinsdale) [reply packages/sky-toolkit-core/settings/_gradients.scss]: If you guys are adding your own mobile gradient called `sky-mobile` in your apps then it should overwrite the value in Toolkit but worth checking.

**Pull request merged**